yt_dlp/extractor/lbry.py

```diff
@@ -67,11 +67,12 @@ def _parse_stream(self, stream, url):
             'release_timestamp': ('value', 'release_time', {int_or_none}),
             'tags': ('value', 'tags', ..., {lambda x: x or None}),
             'duration': ('value', stream_type, 'duration', {int_or_none}),
-            'channel': ('signing_channel', 'value', 'title', {str}),
             'channel_id': ('signing_channel', 'claim_id', {str}),
         })
```

Comment:
Do not remove `channel` here; it makes no sense to do so.
If we want to add `uploader_id`, then let's do it here
```suggestion
            'channel': ('signing_channel', 'value', 'title', {str}),
            'channel_id': ('signing_channel', 'claim_id', {str}),
            'uploader_id': ('signing_channel', 'name', {str}),
        })
```